src/python/WMCore/WorkQueue/DataStructs/WorkQueueElement.py

```diff
@@ -213,7 +213,7 @@ def statusMetrics(self):
     def passesSiteRestriction(self, site):
         """Takes account of white & black list, and input data to work out
         if site can run the work"""
-        # Don't check anything if useSiteListAsLocation/trusSiteLists is enabled
+        # Don't check anything if trustSitelists is enabled
         if self['NoLocationUpdate']:
```

Comment:
It's set in the WorkQueue start policies, e.g.:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WorkQueue/Policy/Start/Block.py#L57

though I just noticed I was mixing an update there as well. Updating the code